packages/pieces/src/apps/cloud-vision/actions/google-auth.ts

```diff
@@ -0,0 +1,19 @@
+import { GoogleAuth } from 'google-auth-library';
+
+function parseServiceAccountCredentials(serviceAccountCredentials: string) {
+  const serviceAccountCredentialsJson = JSON.parse(serviceAccountCredentials);
+  return {
+    client_email: serviceAccountCredentialsJson.client_email,
+    private_key: serviceAccountCredentialsJson.private_key,
+  };
+}
+
+export function createAuth(serviceAccountCredentials: string) {
```

Comment:
Can you we add it inside named export in order to not populate the general scope, for example.

```typescript
export const googleCloudVisionCommon {
   // exported functions goes there
}
```